data/json/recipes/armor/suit.json

```diff
@@ -285,12 +285,13 @@
     "category": "CC_ARMOR",
     "subcategory": "CSC_ARMOR_SUIT",
     "skill_used": "fabrication",
-    "difficulty": 7,
+    "difficulty": 3,
     "time": 100000,
     "reversible": true,
     "autolearn": true,
-    "book_learn": [ [ "recipe_melee", 6 ] ],
+    "book_learn": [ [ "recipe_melee", 2 ] ],
     "using": [ [ "cordage", 2 ] ],
+    "qualities": [ { "id": "CUT", "level": 1 }, { "id": "HAMMER", "level": 1 } ],
     "components": [ [ [ "scrap", 15 ] ] ]
   },
```

Comment:
The weight of scrap required is significantly less than the weight of the scrap suit.

Reply:
Could work.  What about the weight of the strings?